src/lib/parameters/parameters.cpp

```diff
@@ -697,11 +819,26 @@ param_set_internal(param_t param, const void *val, bool mark_saved, bool notify_
 	}
 
 	if (handle_in_range(param)) {
-
 		param_wbuf_s *s = param_find_changed(param);
 
-		if (s == nullptr) {
+		// // don't store new value if default
+		// if (val_is_default) {
```

Comment:
Why is this commented?

Reply:
@dagar I was thinking about one case where this might lead to undesired behavior but thinking about it again I think it's ok. I will add this again.